src/dash-adapter.js

```diff
@@ -7,6 +7,20 @@ import {Error} from 'playkit-js'
 import Widevine from './drm/widevine'
 import PlayReady from './drm/playready'
 
+
+type ShakaEventsType = { [event: string]: string };
+
+/**
+ * Shaka events enum
+ * @type {Object}
+ * @const
+ */
+const SHAKA_EVENTS: ShakaEventsType = {
```

Comment:
Enums convention are to lead with capitals and refer as singular (i.e. `ShakaEvent`)